SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -36,7 +36,7 @@ USpatialGDKSettings::USpatialGDKSettings(const FObjectInitializer& ObjectInitial
 	, bBatchSpatialPositionUpdates(false)
 	, MaxDynamicallyAttachedSubobjectsPerClass(3)
 	, bEnableServerQBI(true)
-	, bEnableClientResultTypes(false)
+	, bEnableClientResultTypes(true)
```

Comment:
```suggestion
	, bEnableClientResultTypes(false)
```

Reply:
will do, running a test to see bandwidth difference on squid with this branch currently so the flag is flipped

Reply:
flipped back, will use other branch for testing bandwidth